src/data/mv_polynomial.lean

```diff
@@ -1292,4 +1323,129 @@ end
 
 end equiv
 
+/-! lemmas about `pderivative v`, the partial derivative with respect to `v : σ`-/
```

Comment:
```suggestion
/-!
## Partial derivatives
-/
```
This will create a section header in the documentation, and the first thing that will follow it is the definition that you give a couple of lines below. I guess most users will understand that some lemmas will follow it.